plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -4114,6 +4115,27 @@ else if (getBukkitEntity() instanceof Creeper) {
             }
         }
 
+        // <--[mechanism]
+        // @object EntityTag
+        // @name loot_table
+        // @input ElementTag
+        // @description
+        // Sets the loot table of a lootable entity.
+        // This is the namespaced path of the loot table, provided by a datapack or Minecraft's default data.
+        // @tags
+        // <LocationTag.loot_table_id>
+        // @Example
+        // # Sets the chest's loot table to a bonus chest
+        // - adjust <player.location.find_entities[zombie].within[5].first> loot_table:entities/phantom
+        // -->
+        if (mechanism.matches("loot_table")) {

```

Comment:
Name for both should have the `_id` suffix (to prevent conflict with any future variant that has better input than a vanilla id, if that happens) (and also to match the tags)